kruize-clowdapp.yaml

```diff
@@ -130,6 +134,10 @@ objects:
               value: "prometheus-k8s"
             - name: savetodb
               value: "true"
+            - name: local
+              value: "false"
+            - name: LOG_ALL_HTTP_REQ_AND_RESPONSE
+              value: "true"
```

Comment:
@khansaad I don't think we need these vars here, i.e. for the type job.

Reply:
Updated now.